Ladybird/Qt/main.cpp

```diff
@@ -159,6 +174,18 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     Ladybird::BrowserWindow window(initial_urls, cookie_jar, web_content_options, webdriver_content_ipc_path);
     window.setWindowTitle("Ladybird");
 
+    singleton_process_state.on_new_tab = [&](auto const& urls) {
+        if (urls.is_empty()) {
+            window.create_new_tab(Web::HTML::ActivateTab::Yes);
+        }
+        for (size_t i = 0; i < urls.size(); ++i) {
+            window.new_tab_from_url(urls[i], (i == 0) ? Web::HTML::ActivateTab::Yes : Web::HTML::ActivateTab::No);
```

Comment:
Not sure whether I should sanitize these urls... this logic matches that in BrowserWindow::BrowserWindow(), but results in the `:` windows at the end of my demo video in the PR description. Unless that is fixed by related the student project PR?

Reply:
(I should, patch incoming)